plans/fix-2805-e2e-installdeps-hang.md

```
@@ -0,0 +1,115 @@
+- `Install Playwright browsers`
+- `Install Playwright OS dependencies` ← 実際に wedge していたステップ
+
+追加するステップ: イメージと `@playwright/test` の**バージョン不一致を検出するガード**。
+片方だけ bump されると 268 テストが1件ずつ「実行ファイルが無い」で落ちるので、その前に落とす。
+
+## 実際に流して分かったこと — イメージにコンパイラが無い
+
+最初は Playwright が文書化している `--user 1001` で組んだが、**両シャードが2分で失敗**した。
+`yarn install` が落ちており、原因は権限ではなく:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)